src/VoltageCalibration.cc

```diff
@@ -1058,64 +1053,51 @@ double * mattak::applyVoltageCalibration (int N, const int16_t * in, double * ou
     return 0;
   }
 
-  bool is2ndHalfWindows;
-  int nwindows = N / mattak::k::radiant_window_size;
-  int isamp;
+  int nSamplesPerGroup = mattak::k::num_radiant_samples;
+  int nWindowsPerGroup = mattak::k::radiant_windows_per_buffer;
   int i = 0;
```

Comment:
I think you do not have to specify `= 0` here (build in datatypes such as `int` has a default initializer which sets the value to 0.  I also think there could a better name ..... ups I just realized you can delete that variable completley right?

Reply:
That `int i = 0` outside of the loop can be removed actually... I will remove it